pkgs/firehose/lib/src/health/license.dart

```diff
@@ -40,3 +40,11 @@ Future<List<String>> getFilesWithoutLicenses(
 Done, found ${filesWithoutLicenses.length} files without license headers''');
   return filesWithoutLicenses;
 }
+
+bool fileIsGenerated(String fileContents) => fileContents
```

Comment:
FYI, another common pattern for a generated file in dart is for it to end in `.g.dart`.